ticket.php

```diff
@@ -881,6 +892,15 @@
                     </div>
                     <!-- End Vendor card -->
 
+                    <!-- Products card -->
```

Comment:
In the main side nav (side_nav.php), we only show products if config_module_enable_accounting is 1. I think this should be the same?